prdoc/pr_5765.prdoc

```diff
@@ -0,0 +1,31 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: ...
+
+doc:
+  - audience: Runtime Dev
```

Comment:
I'd add another description for `Runtime User` saying that now any user account can have a sovereign account on another chain controlled by that first account. This can be useful to have some funds on other chains

Reply:
Added